tests/00_mesh/prism_11el.msh

```diff
@@ -50,3 +50,109 @@ $Elements
 24 4 2 6 201 7 2 1 8
```

Comment:
Move this data file to the folder of the particular test. In 00_mesh we want pure geometric meshes.